src/main/java/org/folio/rest/impl/base/AbstractEmail.java

```diff
@@ -135,21 +134,21 @@ protected Future<EmailEntity> processEmail(EmailEntity email, SmtpConfiguration
 
   protected EmailEntity handleSuccess(EmailEntity email) {
     String message = format(SUCCESS_SEND_EMAIL, join(",", email.getTo()));
-    logger.info(message);
+    logger.info("handleSuccess:: {}", message);
 
     return updateEmail(email, DELIVERED, message);
   }
 
   protected EmailEntity handleFailure(EmailEntity email, Throwable throwable) {
```

Comment:
Same here - we should log method name and params when entering the method and the result on exit